### ISSUE TITLE
Airflow web.baseUrl incorrect if using NodePort

### ISSUE THREAD

**Issue by rajan123456** [bitnami/charts#20573]
### Name and Version

bitnami/airflow 16.1.0

### What architecture are you using?

None

### What steps will reproduce the bug?

1. Install Airflow helm chart, get this warning that web.baseUrl is not set

NOTE: Clipped for brevity
```bash
###############################################################################
### ERROR: You did not provide an external URL in your 'helm install' call ###
###############################################################################

This deployment will be incomplete until you configure Airflow with a resolvable
host. To configure Airflow with the URL of your service:

1. Get the Airflow URL by running:

    export AIRFLOW_HOST=$(kubectl get nodes --namespace ns-airflow -o jsonpath="{.items[0].status.addresses[0].address}")
    export AIRFLOW_PORT=$(kubectl get --namespace ns-airflow -o jsonpath="{.spec.ports[0].nodePort}" services darc-airflow)       

2. Complete your Airflow deployment by running:

###CLIPPED###

    helm upgrade --namespace ns-airflow darc-airflow oci://registry-1.docker.io/bitnamicharts/airflow -f airflow-values.yaml
```

3. Apply the required setting through values.yaml file

```yaml
web:
  baseUrl: http://localhost:31020
```
4. Describe the deployed pod, the value is not set correctly.

```
Environment:
      AIRFLOW_FERNET_KEY:                   <set to the key 'airflow-fernet-key' in secret 'darc-airflow'>  Optional: false       
      AIRFLOW_SECRET_KEY:                   <set to the key 'airflow-secret-key' in secret 'darc-airflow'>  Optional: false       
      AIRFLOW_LOAD_EXAMPLES:                no
      BASH_DEBUG:                           1
      BITNAMI_DEBUG:                        true
      AIRFLOW_DATABASE_NAME:                airflow
      AIRFLOW_DATABASE_USERNAME:            postgres
      AIRFLOW_DATABASE_PASSWORD:            <set to the key 'password' in secret 'darc-airflow-externaldb'>  Optional: false      
      AIRFLOW_DATABASE_HOST:                xxx
      AIRFLOW_DATABASE_PORT_NUMBER:         5432
      REDIS_HOST:                           xxx
      REDIS_PORT_NUMBER:                    6379
      REDIS_PASSWORD:                       <set to the key 'redis-password' in secret 'darc-airflow-externalredis'>  Optional: false
      AIRFLOW_EXECUTOR:                     CeleryExecutor
      AIRFLOW_WEBSERVER_HOST:               0.0.0.0
      AIRFLOW_WEBSERVER_PORT_NUMBER:        8080
      AIRFLOW_USERNAME:                     admin
      AIRFLOW_PASSWORD:                     <set to the key 'airflow-password' in secret 'darc-airflow'>  Optional: false
      AIRFLOW_BASE_URL:                     http://http://localhost:31020:8080
      AIRFLOW_LDAP_ENABLE:                  no
      REQUESTS_CA_BUNDLE:                   xxx
      AIRFLOW__WEBSERVER__X_FRAME_ENABLED:  True
      AIRFLOW__API__AUTH_BACKENDS:          airflow.api.auth.backend.basic_auth,airflow.api.auth.backend.session
      AIRFLOW__CORE__DEFAULT_TIMEZONE:      Asia/Singapore
      AIRFLOW__WEBSERVER__SECRET_KEY:       xxx
```

Issues:
1. Value of AIRFLOW_BASE_URL environment variable is rendered incorrectly, based on the instructions in NOTES.txt
2. The environment variable AIRFLOW_BASE_URL seems deprecated/incorrect, correct variable should be AIRFLOW__WEBSERVER__BASE_URL

### Are you using any custom parameters or values?
```yaml
web:
  baseUrl: http://localhost:31020

service:
  nodePorts: 
    http: 31020
  type: NodePort
```
### What is the expected behavior?

Env variable `AIRFLOW__WEBSERVER__BASE_URL=http://localhost:31020`

instead of 

`AIRFLOW_BASE_URL=http://http://localhost:31020:8080`

### What do you see instead?

`AIRFLOW_BASE_URL=http://http://localhost:31020:8080`

**Comment by rajan123456** [comment]
Happy to work on the fix if the expected behavior is acceptable. Thanks!

**Comment by migruiz4** [comment]
Hi @rajan123456,

I'm sorry for the late response. I have implemented your feature request here: #21028

With those changes, it would be possible to provide `web.baseUrl` including schema and port.
